Content.Shared/Storage/AfterStorageFillEvent.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects;
+
+namespace Content.Shared.Storage
+{
+    [PublicAPI]
+    public sealed class AfterStorageFillEvent : EntityEventArgs
+    {
+        public IEntity Entity;
+        public IReadOnlyList<IEntity>? ContainedEntities { get; }
```

Comment:
don't think this needs to be nullable, it should just pass in an empty List if there's no entities